beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -1667,12 +1694,33 @@ impl<T: BeaconChainTypes> Worker<T> {
                     "type" => ?message_type,
                 );
 
-                // We tolerate messages that were just one slot late.
-                if *message_slot + 1 < *earliest_permissible_slot {
+                // Compute the slot when we received the message.
+                let received_slot = self
+                    .chain
+                    .slot_clock
+                    .slot_of(seen_timestamp)
+                    .unwrap_or_else(|| self.chain.slot_clock.genesis_slot());
+
+                // The message is "excessively" late if it was more than one slot late.
+                let excessively_late = received_slot + 1 < sync_committee_message_slot;
```

Comment:
I think this is backwards? Don't we want:

```suggestion
                let excessively_late = received_slot > sync_committee_message_slot + 1;
```

Reply:
Good catch! :sweat_smile: